SimPEG/utils/io_utils.py

```diff
@@ -309,16 +315,18 @@ def writeUBCmagneticsObservations(filename, data_object):
 
     B = survey.source_field.parameters
 
-    rxLoc = survey.source_field.receiver_list[0].locations
-
-    d = data_object.dobs
-    wd = data_object.standard_deviation
+    data = survey.source_field.receiver_list[0].locations
+    
+    if data_object.dobs is not None:
+    	data = np.c_[data, data_object.dobs]
+    
+    if data_object.standard_deviation is not None:
```

Comment:
Can `data_object.standard_deviation` be None?

Reply:
Yes. I want you to be able to load predicted data as well.